src/dra/menu.c

```diff
@@ -3020,8 +3020,214 @@ bool func_800FB1EC(s32 arg0) {
     return false;
 }
 
-s32 func_800FB23C(MenuNavigation* nav, u8* order, u8* count, u32* selected);
-INCLUDE_ASM("dra/nonmatchings/menu", func_800FB23C);
+typedef enum ItemTypes {
```

Comment:
Is there a difference vs.
```
typedef enum EquipKind {
    EQUIP_HAND,
    EQUIP_HEAD,
    EQUIP_ARMOR,
    EQUIP_CAPE,
    EQUIP_ACCESSORY,
    NUM_EQUIP_KINDS,
} EquipKind;
```
?

Reply:
Nope, this was just the enum that was used in Xeeynamo's old scratch, didn't know there was a different one! I'll update.